graphql/subcourse/mutations.ts

```diff
@@ -371,4 +382,28 @@ export class MutateSubcourseResolver {
         logger.info(`Subcourse(${subcourseId}) was manually promoted by instructor(${context.user.userID})`);
         return true;
     }
+
+    @Mutation((returns) => Boolean)
+    @AuthorizedDeferred(Role.ADMIN, Role.OWNER)
+    async subcourseMeetingJoin(@Ctx() context: GraphQLContext, @Arg('subcourseId') subcourseId: number) {
```

Comment:
Why do we have different mutations for subcourse and match meetings? Why not just appointmentMeetingJoin(...)?

Reply:
Even though the use cases are quite similar at the moment, I recommend keeping them separate since they represent two distinct business scenarios.

Reply:
hm, but in the UI they are all "appointments" so we now anyways need to branch off (currently in the frontend)

Reply:
Interesting. 
What's the expected setup?

Reply:
Well I'd prefer to have appointmentJoinMeeting(...) and then branch off in the backend. But this does not block this PR